sim/target-agnostic.mk

```diff
@@ -74,12 +74,12 @@ $(VERILOG) $(HEADER) $(fame_annos): $(FIRRTL_FILE) $(ANNO_FILE)
 conf: $(fame_annos)
 	mkdir -p $(GENERATED_DIR)
 	cd $(base_dir) && \
-	$(SBT) "project $(midas_sbt_project)" "runMain midas.stage.RuntimeConfigGeneratorMain \
+	$(call run_scala_main,$(firesim_sbt_project),midas.stage.RuntimeConfigGeneratorMain,\
 		-td $(GENERATED_DIR) \
-		-ggaf $(fame_annos) \
+		-faf $(fame_annos) \
```

Comment:
Did this just not get changed in #557 ?

Reply:
I forgot to do the config generator. i should cherry-pick this onto master.